deps.edn

```diff
@@ -1,24 +1,23 @@
 {:paths ["db" "resources" "src"]
 
  :deps
- {org.clojure/clojure {:mvn/version "1.10.0"}
+ {org.clojure/clojure         {:mvn/version "1.10.0"}
   coast-framework/coast.theta {:mvn/version "1.5.0"}
-  org.xerial/sqlite-jdbc {:mvn/version "3.25.2"}}
+  org.xerial/sqlite-jdbc      {:mvn/version "3.25.2"}}
 
  :aliases
  {:test
   {:extra-paths ["test"]
-   :main-opts ["-m" "cognitect.test-runner"]
+   :main-opts   ["-m" "cognitect.test-runner"]
    :extra-deps
    {com.cognitect/test-runner {:git/url "git@github.com:cognitect-labs/test-runner"
-                               :sha "5f2b5c2efb444df76fb5252102b33f542ebf7f58"}}}
+                               :sha     "5f2b5c2efb444df76fb5252102b33f542ebf7f58"}}}
 
   :uberjar
-  {:main-opts ["-m" "mach.pack.alpha.capsule"
-               "-m" "server"
-               "-e" "target"
```

Comment:
Note: this line is removed in the commit to avoid infinite build time.

Reply:
😅